L5-while Loopis/HEX.txt

```diff
@@ -0,0 +1,17 @@
+Dla jasności, już Ci mało czasu zostało, a chcę to jeszcze sprawdzić dzisiaj przed wieczorem...
+Miałeś 2 tygodnie na to, by wrzucić to.
+Wiesz, że zadanie przysyłam Ci we wrzuconych lekcjach, a prosiłem Cię o to na ostatniej lekcji, gdzie powiedziałeś, że zrobisz to "jutro", czyli następnego dnia po lekcji.
+Już ostatnio stwierdziłeś, gdy nie przygotowałeś się do zajęć, że się to więcej nie powtórzy, a znów ma to miejsce, że robisz coś na ostatnią chwilę.
+Pytałem Cię, jak dużo zadania chcesz mieć, powiedziałeś, że tyle to ok i to Ci 5 minut zajeło ostatnio...
+Prosiłęm Cię ponownie o przesłanie plików 24.02, 04.03, 05.03.
+Nie chcę Ci zawać zadania, którego i tak nie zrobisz. Jeśli nie chcesz mieć zadań domowych to powiedz to wprost Filip.
+Szanuj proszę też mój czas i to, że muszę to zadanie Ci tutaj uploadować.
+
+To jednak zadanie dokończ, bo się już do tego zobowiązałeś, jesteś już dorosłą osobą, która jest odpowiedzialna i potrafi się świadomie wywiązywać ze zobowiązań takie mam wrażenie.
+
+Poniżej zadania:
+
+TODO Zrobić notatki
+TODO Zrobić program z pętlą while do zadania z L4
+TODO Dokończyć nasze zadanie
```

Comment:
Przypominam, tak by jeśli liczba jest za niska, użytkownik dostał info, że jest za niska, jeśli odwrotnie to analogicznie.
Jeśli jest już dobra, to gratulacje, udało Ci się odgadnąć za x'tym razem...